src/main/java/com/bitso/BitsoAccountStatus.java

```diff
@@ -42,7 +43,7 @@ public BitsoAccountStatus(JSONObject o) {
         this.signedContract = Helpers.getString(o, "signed_contract");
         this.originOfFunds = Helpers.getString(o, "origin_of_funds");
         this.referralCode = Helpers.getString(o, "referral_code");
-
+        this.cashDepositLimit = Helpers.getString(o, "cash_deposit_allowance");
```

Comment:
Ewww... Why is this a string? Should it be a `BigDecimal` instead?

Reply:
@vogelito I just changed to `BigDecimal`, Initially was a `String` because converting the `String` to `BigDecimal` could be a little bit slow and since we are not going to perform any operations with that value I wanted to keep it simple.

Reply:
Thanks. I'm pretty sure converting the `String` to `BigDecimal` is completely negligible